Packs/DigitalGuardian/Integrations/DigitalGuardianARCEventCollector/DigitalGuardianARCEventCollector.py

```diff
@@ -18,276 +17,274 @@
-        inc_time = event.get("inc_mtime")
-        if last_time:
-            last_time_date = arg_to_datetime(arg=last_time, required=True).date()   # type: ignore[union-attr]
-            event_date = arg_to_datetime(arg=inc_time, required=True).date() if inc_time else None   # type: ignore[union-attr]
-            if (inc_time and inc_time < last_time) or event.get("dg_guid") in id_list:
-                continue
-            if last_time_date == event_date and event.get("dg_guid"):
-                id_list.append(event.get("dg_guid"))
-            elif event.get("dg_guid"):
-                id_list = [event.get("dg_guid")]
-        else:
-            if event.get("dg_guid"):
-                id_list.append(event.get("dg_guid"))
-        event_list_for_push.append(event)
-        last_time = inc_time if inc_time else last_time
-        index += 1
-        if index == limit:
+    event_fields = [field['name'] for field in raw_response['fields']]
+    events_data = raw_response['data']
+
+    events = []
+    for index, event_data in enumerate(events_data):
+        if limit is not None and index == limit:  # No API-side limit param (needs to be managed on our end)
             break
-    return event_list_for_push, last_time, id_list
+
+        event: dict[str, Any] = {}
+        for field, value in zip(event_fields, event_data, strict=True):
+            event[field] = value if value != "-" else None  # "-" mark an empty field in Digital Guardian
+
+        add_time_and_profile_to_event(event, export_profile)
+
+        events.append(event)
+
+    return events
 
 
-def fetch_events(client: Client, last_run: dict[str, list], limit: int) -> Tuple[dict, list]:
+def get_events_command(client: Client, args: dict, export_profile: str) -> tuple[list[dict], dict, CommandResults]:
     """
+    Fetches a limited number of events and returns it in the CommandResults as a human readable markdown table.
     Args:
-        client (Client): DG client to use.
-        last_run (dict): A dict with a key containing the latest event created time we got from last fetch.
-        limit (int):
-
+        client (Client): Digital Guardian client.
+        export_profile (str): Export profile name.
+        args (dict): Command arguments.
     Returns:
-        dict: Next run dictionary containing the timestamp that will be used in ``last_run`` on the next fetch and list of ID's
-              of ingested id's
-        list: List of events that will be created in XSIAM.
+        tuple[list, dict, CommandResults]: List of events, last run dictionary, CommandResults with human readable output.
     """
-    id_list = last_run.get('id_list', [])
-    last_time = str(last_run.get('start_time') or "")
-    demisto.debug('fetching events')
-    event_list = get_raw_events(client, last_time)
-    event_list_for_push, time_of_event, id_list = create_events_for_push(event_list, last_time, id_list, limit)
+    limit = arg_to_number(args.get('limit')) or 1000
```

Comment:
is this `1000` limit documented anywhere?

Reply:
It is documented in the yml, README, and in Confluence. 

Are there any other places you recommend?